File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcRoot/IfcPropertyDefinition/IfcPropertySetDefinition/IfcQuantitySet/IfcElementQuantity/IfcElementQuantity.vb
Public Class IfcElementQuantity Inherits IfcQuantitySet
    Public MethodOfMeasurement As IfcLabel
    Public Quantities As SET [1:?] OF IfcPhysicalQuantity

    ' === WHERE CLAUSES ===
    ' UniqueQuantityNames : IfcUniqueQuantityNames(Quantities)
End Class

File: src/main/resources/output/IFC4X3_csharp/Entities/IfcRoot/IfcPropertyDefinition/IfcPropertySetDefinition/IfcQuantitySet/IfcElementQuantity/IfcElementQuantity.cs
public class IfcElementQuantity : IfcQuantitySet
{
    public IfcLabel MethodOfMeasurement { get; set; }
    public SET [1:?] OF IfcPhysicalQuantity Quantities { get; set; }

    // === WHERE CLAUSES ===
    // UniqueQuantityNames : IfcUniqueQuantityNames(Quantities)
}
